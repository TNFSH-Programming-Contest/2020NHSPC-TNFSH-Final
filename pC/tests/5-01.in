99 1579 1
5 12 838627
24 5 151782
51 24 557929
5 86 170762
86 57 155750
56 57 386159
56 79 414924
57 99 342869
57 58 119758
58 75 789471
90 75 573181
39 90 900655
90 89 778244
89 74 307128
74 8 439783
49 8 331982
26 49 682918
26 3 545194
3 77 22729
49 69 914724
62 74 827960
74 97 862304
71 89 584242
42 71 656834
11 71 911776
32 89 143161
89 66 96272
27 89 500362
19 27 10357
27 80 70467
75 34 233277
34 88 172976
84 75 645839
76 58 666100
4 76 227328
98 58 11360
38 98 184008
38 2 493305
98 78 315409
78 65 288257
60 5 814604
60 68 316298
68 93 816977
60 54 639659
54 73 503707
5 17 655159
28 5 648292
83 28 717422
13 28 11561
28 87 8386
70 5 585939
96 70 753511
48 96 950295
96 52 919101
15 96 301854
82 96 428569
82 81 351151
63 81 873118
96 14 7204
53 14 689650
53 72 800115
16 72 862642
55 16 922571
23 96 303050
5 43 597753
43 67 819953
43 45 108718
20 5 125300
20 85 556771
22 20 613082
22 29 483018
29 1 509067
35 22 809578
30 35 235106
37 35 652278
33 37 69352
44 37 470223
21 44 581030
44 9 632557
47 44 757215
47 6 679734
44 7 552554
44 64 634071
40 64 788001
40 92 649329
92 10 560466
36 10 377152
50 40 748600
50 31 200020
91 40 494467
44 95 185184
18 95 862476
95 25 491572
25 94 290906
46 25 55185
95 41 908751
37 59 419998
59 61 969362
46 92 148881
90 22 93163
55 3 305856
34 59 289806
55 72 517955
59 49 797829
54 39 631196
44 45 395855
2 80 329560
83 70 185942
98 81 620899
44 20 555310
10 40 965659
58 77 408267
43 90 255717
1 64 562059
4 29 424997
61 20 447123
4 17 183999
66 43 133086
53 48 502242
65 30 616393
44 4 380342
88 83 394181
33 80 998981
92 39 246473
95 54 117848
92 74 250133
69 95 811189
30 21 861528
41 14 339592
89 97 471717
71 15 851082
8 29 89873
84 57 878027
13 3 900335
6 12 669013
15 92 903790
57 32 917318
75 10 636553
95 3 38143
74 16 324794
55 67 792245
59 35 126791
54 45 779201
65 45 365182
23 77 874038
99 23 171254
90 25 908624
14 58 413406
47 8 516767
28 84 127780
93 72 250272
42 80 951306
32 67 673601
13 45 868470
57 7 542316
30 39 892231
87 54 451217
25 55 380038
18 36 718865
72 46 382029
98 70 964165
43 75 763363
55 15 537107
85 21 110481
70 47 762740
42 69 79304
16 63 222982
88 17 940673
61 41 448082
74 10 303121
89 88 854399
59 9 871273
12 13 180793
39 73 11064
20 33 985498
56 22 663034
47 89 649688
88 39 157860
81 48 413644
12 47 98522
34 96 330999
45 88 313107
35 14 260441
88 26 725788
79 40 395909
43 16 942492
34 63 683265
43 49 203858
17 63 33755
28 17 398368
17 83 431729
10 73 323938
84 37 726409
25 77 708302
16 37 80940
79 59 131947
99 39 132764
27 2 108558
2 99 983213
83 93 326084
89 58 170996
75 27 773471
75 91 70271
44 85 666858
42 38 80272
61 1 406871
59 72 680651
41 76 491747
91 25 740824
22 12 216152
9 8 811235
88 62 332420
89 56 984997
23 56 642688
82 7 733251
11 45 572995
97 33 267481
83 2 474203
25 88 243076
30 26 898816
31 93 661560
71 1 999488
97 45 883078
7 74 189048
38 53 319705
71 91 350563
22 63 41911
43 44 904513
8 59 840635
94 4 175741
12 35 219308
33 56 719993
7 41 208729
56 44 743726
69 51 516727
31 15 809312
30 73 974443
30 57 377139
91 57 640826
77 36 375967
24 76 145901
87 91 640988
81 49 448152
17 97 97581
40 23 528286
62 15 840139
99 97 106341
94 21 337010
31 97 273124
33 74 331395
55 60 222934
77 59 249093
18 46 647919
39 2 276056
93 30 948685
75 25 143549
56 42 722044
33 25 896086
97 32 719317
74 99 432030
3 57 133984
53 37 188499
26 41 486235
16 9 231628
74 25 635647
95 43 220751
31 45 838455
21 31 322027
70 81 181438
54 1 580778
26 15 229989
87 53 782328
58 78 886385
67 26 401321
55 81 532633
63 43 892746
78 84 110028
85 3 596295
17 74 907808
84 4 538437
99 87 739264
99 75 167498
5 3 265358
98 52 315917
34 85 530317
3 61 290157
22 82 154078
50 86 37478
97 5 307373
51 73 582969
25 16 386040
9 70 642002
21 28 622491
61 8 218249
20 91 395666
62 98 612054
62 26 366443
12 72 118286
70 6 777034
61 78 756419
9 65 542659
57 5 324197
49 60 213846
9 76 234156
21 64 452887
50 46 929909
75 42 973235
95 42 836757
59 87 300090
87 56 560440
46 47 701624
35 28 404026
73 95 635783
15 65 303408
66 41 732754
37 85 632789
35 34 388223
28 42 341440
41 10 482417
53 44 287698
56 4 859501
8 50 352243
80 56 877695
76 62 356484
49 98 72579
76 52 795174
72 84 899974
66 1 73113
93 90 953971
85 10 600711
80 95 69582
6 93 73071
55 75 374072
27 63 386782
77 70 960613
42 23 193447
4 6 317794
56 65 925945
53 10 55000
17 95 747465
97 9 529248
25 78 436657
36 17 344298
62 20 128621
94 80 427159
86 2 548652
60 44 509965
47 85 914991
5 9 139568
2 73 864648
68 43 994357
92 98 874530
98 34 638772
37 51 984797
66 76 383238
37 55 812489
72 36 109998
50 30 767030
10 72 421923
37 45 963605
54 2 966019
93 18 98877
83 19 54186
54 14 184535
46 4 632398
73 80 706040
35 55 124988
41 53 13497
63 41 864004
37 38 433242
1 89 851884
6 63 606317
35 46 857139
27 21 956213
84 97 832509
58 66 211234
75 92 774716
70 39 663826
80 36 235184
43 47 571312
59 58 420676
1 46 304660
30 42 803934
88 59 385533
77 68 92694
8 77 577105
9 38 114185
55 53 861388
15 21 446197
7 80 702896
1 57 987049
22 15 519558
47 66 697830
23 9 542415
54 7 611485
22 67 661082
58 6 201035
8 53 571874
34 67 499540
83 22 247105
22 74 814502
95 77 494034
43 20 331369
62 56 144188
45 1 491315
56 25 644614
75 94 12019
94 5 847740
50 68 166192
38 17 352282
64 19 852735
13 39 54505
37 99 729814
65 70 44207
9 87 739290
73 22 361180
17 13 584268
94 68 752607
1 31 929820
24 39 295950
10 54 298659
22 85 69461
52 89 411064
40 48 734848
65 60 615189
45 78 12215
20 28 207835
33 79 377661
75 80 977416
52 8 756443
55 80 261885
69 40 402948
70 28 122032
62 19 71392
29 43 696055
53 83 344236
63 59 221460
9 17 957382
35 42 350895
73 87 674250
49 15 178455
65 26 849944
17 15 279272
13 51 643371
2 28 350609
19 75 766037
91 38 772148
6 60 172958
67 28 83415
93 15 128781
51 92 295410
14 10 652537
55 61 454519
11 16 951152
1 68 429814
14 89 583844
72 4 228646
99 26 691636
18 25 685561
19 80 908826
82 85 589330
20 17 190938
9 32 474266
61 87 617665
98 15 903445
69 27 265406
97 35 418041
84 77 212087
62 85 876491
12 49 8586
1 94 571693
73 1 598374
51 56 325866
93 22 489427
59 31 930137
48 69 449122
65 97 873274
59 55 2485
28 32 263333
83 23 649795
85 69 270810
19 40 610726
20 79 78208
56 49 228506
97 59 279995
32 27 204736
70 86 234114
25 96 241219
24 65 348058
83 1 230533
56 39 76462
35 75 2965
80 79 341570
24 92 981639
70 29 629827
72 28 860604
78 2 556317
47 23 767784
95 8 524180
25 61 538915
23 64 120396
7 47 801051
27 9 192173
51 91 895830
14 49 159089
97 62 56063
65 90 522689
42 59 50102
9 4 20071
16 4 710783
7 86 529957
42 67 872247
16 83 503021
17 96 163627
38 58 436877
39 19 468359
33 22 841615
74 72 988139
50 20 104463
9 86 27692
23 11 242630
74 27 642301
88 29 664715
59 20 554850
53 65 768308
10 38 873380
95 86 835436
99 49 333858
60 48 239689
18 49 144197
9 26 382494
37 78 221955
72 14 420998
77 40 578364
77 57 787336
90 23 309614
99 77 928923
20 7 79324
44 89 457131
74 77 521852
93 29 659557
74 32 962969
17 93 573604
60 12 524029
85 49 925845
63 21 684914
80 88 98841
67 5 45937
33 54 632093
22 39 185771
85 56 343929
84 34 52699
51 40 524720
4 27 997038
42 88 978883
89 23 695644
64 39 804023
32 62 985148
16 8 767719
82 2 373159
84 42 526625
11 18 928026
76 84 67414
1 8 82923
6 18 588626
4 93 76833
91 22 749940
31 16 823249
31 29 157306
22 89 325944
4 47 316606
56 10 851643
52 42 997864
49 77 114795
16 89 805945
62 6 730515
55 96 806357
41 6 526265
47 26 655052
57 64 453321
7 56 481805
37 60 908908
2 10 208886
11 59 790449
60 52 819159
5 46 483547
16 30 602094
80 51 68241
45 74 83200
72 23 486981
10 34 508531
98 94 735483
33 57 671786
48 51 913066
27 1 655996
24 23 6674
91 41 269928
71 78 137839
78 56 750167
51 52 464614
95 50 674007
5 76 998983
96 46 591435
79 58 10666
53 3 403660
40 63 746549
61 49 670283
27 13 445431
96 41 918533
56 21 42991
44 12 431468
42 57 649789
71 34 744993
21 37 800217
17 42 676394
74 93 784338
39 5 475568
67 53 306742
67 93 948730
19 26 133717
66 98 299930
42 24 468582
96 47 668626
79 69 310421
36 69 476830
91 30 20918
65 36 108739
90 95 360450
36 43 912753
15 20 668592
78 1 56047
2 32 481771
52 5 816882
83 41 580863
58 99 527469
77 94 234629
38 99 543263
98 3 335022
34 26 503254
98 1 279968
88 49 467497
44 87 197366
5 50 432760
91 8 652811
48 95 116738
44 46 737826
54 65 576598
77 37 176920
39 50 817776
54 83 513388
27 64 986669
50 70 301971
13 73 720064
10 39 435131
63 67 976741
30 51 908788
57 36 985792
51 75 324623
28 75 537403
53 11 583776
87 69 122203
63 98 335125
17 2 282197
46 67 914581
13 16 705962
5 2 141134
14 85 642139
50 76 108500
14 48 526555
40 73 5175
42 37 2958
21 77 984598
23 70 957913
33 98 360372
4 67 692111
29 34 637179
11 75 761188
84 56 413656
60 77 245409
40 97 218477
56 83 89220
65 73 704417
33 40 895783
27 60 12427
46 13 144127
23 78 569644
73 32 793888
62 44 391279
89 59 849559
1 44 518116
67 87 715405
19 4 180022
12 73 46303
17 61 489722
17 86 83430
8 68 564662
50 22 202280
57 94 61248
6 64 857343
42 5 211164
21 92 981973
67 50 781185
24 60 746699
87 83 72557
70 92 22118
81 40 194581
26 25 304258
5 41 794814
47 52 8591
42 82 315233
99 34 761034
29 79 34140
71 80 694574
23 16 343161
79 85 398978
29 47 991959
77 1 92465
28 45 448556
55 68 552775
24 29 231350
82 70 62072
20 84 369134
21 88 38662
9 54 4718
16 84 139077
37 73 193282
23 67 243061
12 95 556628
15 59 167784
12 19 624484
7 96 17891
84 33 102607
88 68 39227
75 26 912396
65 68 562840
53 73 695313
2 3 408985
94 76 831975
83 14 247357
84 81 529850
43 35 594204
12 28 215650
63 23 529244
80 43 549841
88 53 643488
79 17 804726
97 36 958879
17 67 873317
19 58 987544
8 26 138349
62 14 896933
93 2 675355
10 78 908686
73 57 149207
65 28 437779
76 34 53884
89 45 196943
97 37 722676
27 46 190367
16 12 259397
63 69 894707
1 87 160358
78 54 904095
41 62 417557
79 60 672177
68 25 98735
67 14 41479
99 6 105256
91 54 662360
69 66 252322
71 8 548201
20 56 419183
46 9 740005
11 12 449403
2 12 731215
24 8 868909
53 15 134346
55 52 537619
92 99 552035
87 40 333355
69 13 18085
68 71 320315
33 89 778702
92 59 223530
43 42 254928
61 22 397256
35 54 249847
64 79 858389
55 12 332075
4 74 750147
55 24 69065
97 98 507914
68 17 978967
92 96 438237
84 43 669664
32 81 243350
53 24 809358
36 61 437080
83 18 90384
23 71 12453
55 99 291658
60 1 273218
55 14 185349
40 31 427737
74 5 409142
63 84 748589
41 70 701142
40 28 922955
30 52 667487
92 30 229196
79 65 272798
67 99 343742
71 29 424919
46 22 738509
61 14 820158
52 70 946416
21 74 702088
9 93 795128
57 19 533751
80 61 399593
52 93 289556
30 56 179918
82 88 779946
75 93 102859
39 23 242177
87 68 971980
18 58 742553
64 13 431616
34 97 468997
28 82 73534
57 80 966064
6 9 65517
70 4 198364
4 59 798993
25 59 804883
26 20 140832
3 32 718583
43 76 541909
80 77 278864
32 69 264659
13 20 758765
30 70 223608
74 37 819958
28 29 979218
72 54 485374
24 22 632265
95 34 131258
96 94 180540
16 48 48553
53 36 784998
19 54 986949
35 82 96522
41 11 719524
46 21 958222
72 48 328559
13 86 992558
91 23 434305
17 32 316720
3 40 713423
93 44 699078
93 13 849233
44 68 335410
31 30 230361
45 6 464873
46 2 249024
50 71 650822
68 51 174581
87 76 712251
98 44 9140
50 26 664175
87 21 659963
77 97 807929
87 89 26946
46 34 999008
57 62 56729
58 60 917917
76 47 263737
16 34 445024
77 76 648415
48 18 825246
72 1 64712
80 67 958264
96 2 948177
61 53 870372
15 19 239046
9 60 315893
62 80 892899
35 9 400471
30 82 658525
75 73 268795
13 78 414767
72 73 282019
14 24 516620
42 60 100256
30 43 804717
31 11 546373
22 58 545581
74 63 990224
48 68 796856
45 87 634982
41 47 937663
21 97 243416
78 89 765633
13 83 427947
38 3 511866
50 6 769135
21 35 217600
90 53 712653
67 19 1156
52 90 944495
21 72 6811
66 27 893319
1 88 488512
16 73 857381
65 92 618525
71 7 902806
5 85 843401
9 15 883465
3 76 903410
20 63 410895
57 44 832191
86 83 660838
74 65 188539
64 62 261369
98 42 617436
18 15 879783
16 35 580908
10 63 19621
64 83 399506
60 43 761163
39 16 996304
33 85 791005
65 83 37371
23 80 438814
17 18 456658
33 91 14091
43 13 171858
3 64 118729
82 89 157469
16 14 980791
73 8 88476
94 37 420310
43 78 679241
52 69 297746
1 59 991079
63 93 582071
47 77 163752
44 28 929676
45 46 751994
25 84 897614
29 37 8410
50 41 645092
5 55 971802
49 36 133851
47 93 433892
8 14 949296
70 1 152689
76 44 855103
6 22 252715
74 6 843618
41 52 567027
60 7 678048
98 87 129578
15 34 521227
29 12 724563
68 62 499732
63 76 156651
10 46 827597
84 55 681672
89 68 135069
75 49 85051
54 37 729235
66 44 25551
52 66 566156
55 73 950308
88 22 578033
13 58 425887
56 67 849124
81 29 992032
71 86 261771
11 47 800725
29 54 966192
18 34 359088
99 51 139385
52 73 845987
82 72 715954
67 15 654090
56 70 26059
61 34 768148
54 17 147819
13 68 951860
73 23 538333
11 8 586027
49 37 500908
9 42 620525
90 45 510034
17 76 255757
15 95 891352
19 11 880655
34 58 155119
99 45 397158
15 87 787642
48 52 525241
92 81 762030
85 13 779376
41 42 636932
8 7 824869
35 74 811565
58 36 363824
19 10 310030
66 57 712115
26 81 829490
99 82 143861
56 9 162034
24 59 208930
13 62 301294
3 89 594066
60 30 659510
6 95 353118
81 16 790358
39 21 172894
97 23 412856
30 80 616423
95 82 678859
61 10 161484
48 45 479808
1 21 626171
86 32 298092
50 64 923765
82 40 255837
6 35 620458
10 7 742046
13 9 784185
64 41 169194
93 51 147227
91 53 883930
54 67 687974
62 36 914982
8 3 582209
41 79 739975
40 45 833266
32 23 216365
3 94 714126
89 36 647435
18 7 524938
61 31 572453
81 22 778992
92 49 570182
40 60 48809
3 78 404149
53 25 260148
24 4 829428
4 79 213787
56 31 437325
49 89 797599
64 92 631390
30 24 220922
95 58 20211
73 5 166096
93 43 370351
10 90 425408
31 3 975807
6 26 268721
66 5 475658
78 52 696592
50 53 557577
44 42 231099
48 90 108007
32 44 113400
5 49 150469
85 52 93804
62 96 867110
99 11 582009
41 77 727322
80 89 983878
3 35 609480
70 2 185458
37 71 219579
35 86 286721
62 99 261948
48 1 615005
66 72 910203
65 76 397580
49 11 609367
90 73 769603
21 33 84611
63 33 568272
6 44 498407
44 41 959045
38 93 868933
50 7 17665
48 31 500433
20 40 529324
64 5 139760
91 44 572518
35 27 125599
54 77 472629
90 14 57831
29 62 799022
6 19 148270
31 54 585890
41 71 239168
70 12 641123
69 70 49798
8 17 574237
25 14 737721
23 3 392561
43 37 839525
59 28 437199
37 66 718912
95 87 791875
11 32 297704
87 71 64207
87 60 989770
48 75 279660
86 59 226346
58 5 725813
94 91 876973
14 22 1142
31 7 988685
71 77 926862
91 48 50167
8 43 353519
56 55 552341
43 7 781473
62 31 778177
89 4 318977
89 46 96865
2 56 388363
97 92 966909
67 25 799702
52 34 283503
11 91 229814
11 66 192008
18 20 334675
65 43 151909
28 56 346921
92 8 520146
99 15 397101
38 61 431476
83 72 410060
70 79 834679
41 73 155460
13 97 370349
19 53 673402
41 3 959265
13 99 595912
89 57 923480
95 33 599354
56 90 803411
61 50 904042
5 35 821369
39 60 268088
76 81 375771
61 97 441831
85 12 445176
23 10 909790
51 28 966660
46 26 82315
63 12 90543
17 58 252210
97 12 188727
75 8 485039
22 55 721281
70 48 700853
98 32 367422
57 26 978429
49 21 431516
73 99 651769
45 94 295112
29 74 125957
42 94 11659
66 18 936525
13 94 931687
96 63 611561
80 70 876191
16 24 320564
5 37 243761
5 4 71164
82 17 335849
98 77 19757
69 41 154949
2 74 496235
95 56 786101
40 99 49735
30 19 175092
88 70 886942
12 92 267796
81 89 171065
42 66 522302
62 89 305205
18 31 874197
43 88 986265
90 97 110134
64 94 311475
52 18 617044
23 45 273337
97 75 406569
28 1 846191
77 65 671457
54 68 344554
97 27 668980
35 13 590161
53 42 19758
77 9 746173
83 27 727459
23 58 901770
66 8 450221
74 15 794875
96 75 331247
64 59 187307
35 81 398120
75 33 440512
22 7 157326
33 58 681158
33 35 317954
53 47 598646
31 25 603513
33 8 147761
17 16 164949
23 92 88070
75 53 216020
76 1 13909
33 30 568112
58 82 267141
75 39 207003
39 74 600627
47 83 70255
67 75 81619
13 19 620115
81 56 286592
34 68 94695
9 52 961753
62 1 448299
30 68 166180
3 69 918583
86 77 632134
66 90 21400
34 53 676633
4 53 176158
73 46 194362
35 1 245890
12 14 271749
89 18 806694
82 29 131280
6 73 784621
38 83 159171
88 97 660427
64 61 464907
43 62 733225
37 19 299342
69 56 463911
27 54 884868
92 33 198499
89 55 290458
63 55 155258
48 61 175347
81 43 769130
89 79 695944
62 9 369247
21 22 381140
64 2 78326
55 65 952747
19 56 870597
79 54 634896
31 69 537735
20 64 543111
63 38 577939
7 55 999236
32 18 933252
6 10 656695
98 21 41721
81 77 667272
52 14 598143
66 65 26699
71 83 433159
14 46 202699
9 57 154506
86 36 654423
95 7 127623
9 53 528871
51 39 796274
18 14 55308
35 78 749069
64 53 682281
31 10 800074
37 64 571111
48 64 579539
5 69 997274
55 74 284667
38 67 827493
14 64 675280
16 71 737242
65 2 646857
55 10 512444
44 30 425749
34 49 974370
74 18 944106
42 86 499707
6 53 455865
99 29 219421
76 68 834285
90 58 801467
30 5 83834
30 36 848211
30 97 999244
44 23 148759
72 6 708799
23 22 247025
18 41 730522
46 66 636270
40 37 705246
34 25 197044
34 47 210474
18 27 720491
91 64 82599
88 10 458143
25 11 112044
22 17 681571
64 77 753189
20 66 53624
71 92 30827
61 52 885289
56 74 808347
71 57 177691
1 2 676152
53 46 276247
40 6 8466
49 47 196724
10 33 662788
76 30 595076
66 15 871479
79 49 50720
37 56 138210
47 56 603781
74 69 440472
58 4 628317
2 29 118388
2 87 514084
35 61 5799
67 61 942967
29 94 884151
99 65 65347
91 18 799962
28 11 378470
58 8 707211
8 12 789432
13 79 596917
4 69 658982
66 74 53168
18 30 797000
3 4 888728
83 97 481827
44 88 522697
40 22 633781
24 97 522122
37 76 321537
14 77 304788
59 96 654070
73 50 417239
10 62 453611
92 80 155178
25 93 548253
15 30 324571
14 93 796084
87 38 571931
52 54 642569
97 60 338749
75 17 352020
22 77 951368
39 47 919940
73 71 670802
70 49 960189
52 13 472240
84 82 305281
33 42 393251
2 37 304625
94 60 29570
84 35 542154
60 45 483383
69 89 512707
24 27 855058
64 70 490114
8 88 733920
91 58 471818
25 58 900552
40 8 538630
58 32 400477
19 73 285220
11 61 671898
43 54 853123
9 25 713428
67 18 516951
58 24 713965
53 40 789888
93 54 676362
18 80 939938
75 57 328782
75 87 762580
52 22 959791
65 75 33786
20 54 480336
21 66 589456
22 41 945571
96 90 267853
83 57 699696
17 50 642965
64 46 294892
6 82 377574
82 86 520639
16 65 438860
43 18 402220
42 58 874665
61 81 171547
12 45 338645
59 47 899062
56 3 166985
81 2 999459
74 26 597197
38 33 281477
63 83 632183
21 2 48577
50 49 280890
90 49 799508
47 86 630505
32 91 750669
51 90 327831
45 42 11482
38 5 831168
11 74 84092
47 14 761941
25 79 317960
42 63 332968
9 36 227910
10 3 464800
92 69 549882
86 65 487747
40 75 918383
3 44 208129
55 32 587580
77 87 526614
50 93 127107
49 71 890259
66 31 865683
93 71 784117
92 17 753523
73 79 708029
21 48 903285
36 47 843103
31 75 866270
71 38 63207
17 53 672935
45 22 610193
29 97 701155
15 86 691952
92 68 321930
8 20 226411
62 48 848469
73 92 704453
22 92 377207
16 79 855053
99 56 383717
84 62 141324
11 46 899081
9 11 794355
63 29 749511
45 73 800701
66 38 539147
68 16 549237
67 85 56860
41 81 545479
83 36 677286
93 8 232228
23 27 253315
98 37 480468
55 21 876713
87 10 18132
2 50 467664
68 57 530979
89 17 90847
53 66 776159
48 79 62008
29 60 284337
18 42 883867
9 90 625319
8 18 767264
77 61 509460
94 65 853321
75 30 665116
47 55 229233
84 60 207885
90 76 205223
78 47 412958
89 26 367084
94 58 633129
76 80 10370
27 12 553268
73 77 143420
4 41 210840
22 62 962311
65 38 589757
66 71 128778
44 58 717771
24 11 825913
41 59 735020
43 71 854709
21 78 864627
74 48 324835
12 24 666715
65 93 554318
66 85 816669
41 78 549660
70 22 754897
78 44 720165
25 37 890179
53 98 152787
71 64 215978
5 11 295242
67 8 618635
27 98 746764
39 78 227809
18 92 83428
38 35 573316
58 37 610248
65 40 743791
98 55 678499
97 3 442108
88 67 788126
28 24 892583
14 66 88697
86 76 108644
52 10 804328
74 24 201431
64 67 327567
65 11 783565
95 21 480482
1 42 516477
84 17 252410
51 21 447911
47 87 68915
79 34 721979
42 55 608132
51 50 677276
15 5 935115
9 66 911863
37 46 200247
32 85 233339
28 86 415631
36 76 925239
48 83 58044
64 96 15085
86 18 454850
8 86 57253
80 97 271215
90 13 4917
24 68 805496
70 94 96603
53 74 413947
4 40 152124
57 12 583342
11 69 811526
77 24 465125
34 87 495591
31 86 26827
70 58 901732
10 45 2489
89 98 827914
37 10 273578
36 41 531287
24 83 964042
35 20 376968
81 11 529827
20 36 387363
77 51 971886
6 32 81078
57 4 850454
88 50 160134
52 1 721549
42 2 648173
