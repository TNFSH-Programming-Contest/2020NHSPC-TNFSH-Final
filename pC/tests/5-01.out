0 360300 230388 482474 756111 740160 420222 82923 450735 212253 530242 365996 298290 323620 1157145 661200 539332 641834 420412 253474 143046 488856 499452 319620 719690 442544 284238 337488 345592 752594 448140 636192 461368 135586 245890 349420 330808 640064 547389 529722 564995 239232 412398 197328 136524 304660 458546 577154 414520 244818 277560 713592 398008 580778 661655 343542 747534 370746 519384 273218 406871 448299 341120 562059 199624 73113 531295 370318 404974 152689 387772 64712 342798 252562 400794 13909 92465 56047 536992 48558 668254 429522 230533 162646 546012 245106 160358 330555 338770 189026 680900 349614 629968 365612 281583 507042 613941 224444 495477 
